pages/osx/open.md

```diff
@@ -6,9 +6,13 @@
 
 `open {{file.ext}}`
 
-- Run a graphical MacOSX application:
+- Run a graphical macOS application:
 
-`open /Applications/{{Application.app}}`
+`open -a {{Application}}`
+
+- Run a graphical macOS app based on the bundle identifier:
```

Comment:
@agnivade Using the bundle id is better since you aren’t making assumptions about where the app is actually located, so is much more reliable for use in say, scripts.

I also included the simpler `-a` example for common/quick command line use… again, hard-coding file paths should be avoided if possible.